### ISSUE TITLE
Feature request: Add method to setting threshold date from data entry form

### ISSUE THREAD

**Issue by aubreyz** [ransome1/sleek#289]
Little more to be said than the title. If sleek uses threshold dates (which it does and should do) suggest make an easy way to enter those (in parallel with due dates which I don't use at all in fact)!

**Comment by dlaidig** [comment]
This looks like a duplicate of #234, right?
